ModularTegustation/lc13_obj/_adventure_console/adventure_layout.dm

```diff
@@ -456,7 +640,7 @@
 	var/list/new_events = events - event_options
 	for(var/_option in new_events)
 		var/datum/adventure_event/A = _option
-		if(initial(A.require_abno))
+		if(initial(A.require_abno) && !all_abnos)	//Do you require an abno and do you NOT have all abnos?
```

Comment:
Im going to look at this a bit more because this should work to make all events appear.

Reply:
I checked the code and this does work to unlock all events. It may not be obvious at first though due to there being 30+ events.